src/python/era_5g_object_detection_common/era_5g_object_detection_common/face_detector.py

```diff
@@ -35,12 +37,12 @@ def __init__(self, **kw):
             raise ImageDetectorInitializationFailed(f"Failed to initialize detector, {MODEL_FILE} does not exist!")
         self.detection_cascade = cv2.CascadeClassifier(MODEL_FILE)
 
-    def process_image(self, frame):
+    def process_image(self, frame: np.array):
```

Comment:
There are missing annotations for return values.

Reply:
ok, I added them